src/main/java/GoToJailUserCase/GoToJailInputData.java

```diff
@@ -0,0 +1,17 @@
+package GoToJailUserCase;
+
+public class GoToJailInputData {
+    boolean injail;
+
+    public GoToJailInputData(boolean jail) {
```

Comment:
This use case requires no user input

Reply:
if I have no input for this class then it would be hard to implement interactor

Reply:
You can take a look at what I wrote, my use case doesn't have input data either